README.md

```diff
@@ -197,6 +197,12 @@ $ brew tap burntsushi/ripgrep https://github.com/BurntSushi/ripgrep.git
 $ brew install ripgrep-bin
 ```
 
+If you're a **MacPorts** user, then you can install ripgrep from the [official ports](https://www.macports.org/ports.php?by=name&substr=ripgrep):
```

Comment:
Could you please wrap this to 79 columns inclusive? If the URL is too long, then just put the `[...](...)` on its own line.